pkg/api/pod/util_test.go

```diff
@@ -4204,3 +4204,166 @@ func TestValidateAllowSidecarResizePolicy(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateInvalidLabelValueInNodeSelectorOption(t *testing.T) {
```

Comment:
I'm not sure if this test should also be cherry picked to v1.32. This came up when resolving the merge conflicts and I thought I'll ask here.

Reply:
no, just add in the tests from the picked PR

Reply:
I have removed the unwanted test.